README.md

```diff
@@ -2,7 +2,11 @@
 
 # My Portfolio
 
-> This is a Microverse project personal portfolio.
+> This is a Microverse project personal portfolio built using HTML and CSS.
+
+## View The Project here
+
+ https://adanzeakonobi.github.io/My-Portfolio-project-2/
```

Comment:
[ Optional Changes ]
- [ ] I think it would be nice you update the way your live demo link shows in your README.md file. You can use any of the two ways below: 👍

`[Live Demo Link](https://adanzeakonobi.github.io/My-Portfolio-project-2/)`
** OR **
`<a href="https://adanzeakonobi.github.io/My-Portfolio-project-2/">Live Demo Link</a>`

Reply:
Acknowledged.
Thank you.